UIKitLesson1/PlistVC.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ViewController.swift
+//  UIKitLesson1
+//
+//  Created by Kirill Dev on 06.12.2022.
+//
+
+import UIKit
+
+final class PlistVC: UIViewController {
+    
+    private var songs = [Song]()
+    
+    @IBOutlet weak var firstButton: UIButton!
```

Comment:
```suggestion
    @IBOutlet private var firstButton: UIButton!
```

все сделать стронгами и приватами